Lesson11/merve/index.html

```diff
@@ -0,0 +1,46 @@
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <nav id="navbar" class="nav">
+  <a href class="a--nav-selector" id="profile-link" target="_blank"> ABOUT</a>
+  <a href="project-section" class="a--nav-selector">WORK</a>
+  <a href="contact-section" class="a--nav-selector">CONTACT</a>
+  </nav>
+  <body>
+  <main id="welcome-section" class="welcome-section">
+  <h1 class="welcome-section-header">Hello I am Merve</h1>
+  <p class="welcome-section-header-info">a frontend developer</p>
+  </main>
+  <div id="project-section" class="project-section">
+  <h2 class="project-section-header">There are some of my projects</h2>
+  <div class="projects-section-content">
+  <a href="" class="btn project"><img class="project-image"/></a>
```

Comment:
If you want to use an image element, make sure to actually add an `src` attribute to it.